docs.yml

```diff
@@ -0,0 +1,16 @@
+version: '2'
+
+services:
+  bdocs:
+    build:
+      context: .
+      dockerfile: ./Dockerfile-dev
+    volumes:
+      - .:/usr/src/app/
+    command: make -C docs html
+  vdocs:
+    image: nginx
+    ports:
+      - '33333:80'
+    volumes:
+      - ./docs/build/html:/usr/share/nginx/html
```

Comment:
Pushed this by mistake in this commit. @TimDaub whenever you wish, I can rebase to split the commit into two.